src/search/overlays.js

```diff
@@ -4,7 +4,6 @@ import { trackEvent } from "./tracking.js";
 
```

Comment:
This is an exact copy of the overlays.js file in coprocure.us so you only have to review one of these.